app/assets/javascripts/discourse/tests/acceptance/composer-editor-mentions-test.js

```diff
@@ -144,4 +151,19 @@ acceptance("Composer - editor mentions", function (needs) {
       ["foo", "user_group", "user", "user2"]
     );
   });
+
+  test("shows users immediately when @ is typed in a reply", async function (assert) {
+    await visit("/");
+    await click(".topic-list-item .title");
+    await click(".btn-primary.create");
+
+    const editor = query(".d-editor-input");
+
+    await simulateKeys(editor, "abc @");
+
+    assert.deepEqual(
+      [...queryAll(".ac-user .username")].map((e) => e.innerText),
```

Comment:
our `queryAll` helper is using jQ so I'm trying to slowly phase it out

```suggestion
      [...document.querySelectorAll(".ac-user .username")].map((e) => e.innerText),
```

Reply:
Thanks!